scenarios/long.yml

```
@@ -7,8 +7,9 @@ name: Long-Running Network
 # The duration of the scenario's runtime, in seconds.
 duration: 600
 
-# The number of validator nodes in the network.
-num_validators: 1
+# Initial validator nodes in the network.
+validators:
+  - name: validator-1
```

a thought: If the `valdiator` name gets a default, then this would be
```suggestion
validators:
  - instances: 1
```
Or it might even be omitted if you define an empty list in the config to be defaulted to a single validator.